integration-tests/auth/package.json

```diff
@@ -2,14 +2,18 @@
   "name": "test-auth",
   "version": "0.0.0",
   "private": true,
+  "prisma": {
+    "seed": "ts-node --compiler-options {\"module\":\"CommonJS\"} prisma/seed.ts"
+  },
   "scripts": {
     "start:dev": "pnpm run prisma:start && next dev",
     "test": "pnpm run prisma:start && vitest run",
     "test-watch": "vitest",
     "start": "next start",
     "lint": "next lint",
     "clean": "rm -rf .turbo && rm -rf node_modules && rm -rf .next",
-    "prisma:start": "prisma generate && prisma migrate deploy",
+    "prisma:start": "prisma generate && prisma migrate dev && pnpm run prisma:seed",
+    "prisma:seed": "prisma db push --force-reset && prisma db seed",
```

Comment:
If we add back `prisma.$reset()` we only need to run `prisma db seed`

Reply:
okay @dillonraphael, I shall update it

Reply:
@dillonraphael I have reverted the changes. I shall try it again taking you PR as reference

Reply:
I have removed the prisma:seed command as when I tested locally, the same seed command ran twice which caused the failure.

Reply:
@siddhsuresh I don't understand. So where is the seed command being called now?

Reply:
@dillonraphael here https://www.prisma.io/docs/guides/database/seed-database#integrated-seeding-with-prisma-migrate
I was just reading the prisma docs on seeding and found that the prisma migrate dev also runs the seed command defined in the package.json

Reply:
@siddhsuresh Ok makes sense. So last question.

Why `prisma db push --force-reset` instead of `prisma migrate reset`?

Reply:
> 

I have removed ```prisma db push --force-reset``` in the latest commit and I am adding the ```prisma migrate reset --force``` in a new commit now to make sure the database does not have any data before the test

Reply:
@dillonraphael any idea why it's failing? It is mentioning some missing packages in the host OS!